test/evergreen.yml

```diff
@@ -195,7 +195,7 @@ functions:
           DEFINED_EVERGREEN_CONFIG_FLAGS=${DEFINED_EVERGREEN_CONFIG_FLAGS/\-DHAVE_BUILTIN_EXTENSION_ZSTD=1/}
         fi
 
-        if [ "${build_variant|}" = "macos-1100" ]; then
+        if [ "${build_variant|}" = "macos-1100" -o "${build_variant|}" = "macos-1100-arm64" ]; then
```

Comment:
what do you think about converting into a regex and look for only macos-1100 here?

Reply:
Thanks, good call - done in 933838f.